[2022-07-08 02:36:34,897] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: client_processing_db.create_customer scheduled__2022-07-07T00:00:00+00:00 [queued]>
[2022-07-08 02:36:34,920] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: client_processing_db.create_customer scheduled__2022-07-07T00:00:00+00:00 [queued]>
[2022-07-08 02:36:34,923] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-08 02:36:34,926] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-08 02:36:34,927] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-08 02:36:34,957] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): create_customer> on 2022-07-07 00:00:00+00:00
[2022-07-08 02:36:34,968] {standard_task_runner.py:52} INFO - Started process 11459 to run task
[2022-07-08 02:36:34,976] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'client_processing_db', 'create_customer', 'scheduled__2022-07-07T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/client_processing_db.py', '--cfg-path', '/tmp/tmpkde14mc2', '--error-file', '/tmp/tmp3jrg1hll']
[2022-07-08 02:36:34,978] {standard_task_runner.py:80} INFO - Job 26: Subtask create_customer
[2022-07-08 02:36:35,150] {task_command.py:369} INFO - Running <TaskInstance: client_processing_db.create_customer scheduled__2022-07-07T00:00:00+00:00 [running]> on host 3e3038753a11
[2022-07-08 02:36:35,639] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=client_processing_db
AIRFLOW_CTX_TASK_ID=create_customer
AIRFLOW_CTX_EXECUTION_DATE=2022-07-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-07T00:00:00+00:00
[2022-07-08 02:36:35,642] {mysql.py:82} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS customers (
                users_id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL UNIQUE,
                password TEXT NOT NULL,
                email TEXT NOT NULL UNIQUE
            );
        
[2022-07-08 02:36:35,678] {base.py:68} INFO - Using connection ID 'mysql' for task execution.
[2022-07-08 02:36:35,712] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/mysql/operators/mysql.py", line 84, in execute
    hook.run(self.sql, autocommit=self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 181, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/mysql/hooks/mysql.py", line 166, in get_conn
    return MySQLdb.connect(**conn_config)
  File "/home/airflow/.local/lib/python3.7/site-packages/MySQLdb/__init__.py", line 123, in Connect
    return Connection(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/MySQLdb/connections.py", line 185, in __init__
    super().__init__(*args, **kwargs2)
MySQLdb._exceptions.OperationalError: (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
[2022-07-08 02:36:35,791] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=client_processing_db, task_id=create_customer, execution_date=20220707T000000, start_date=20220708T023634, end_date=20220708T023635
[2022-07-08 02:36:35,854] {standard_task_runner.py:97} ERROR - Failed to execute job 26 for task create_customer ((2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)"); 11459)
[2022-07-08 02:36:35,919] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-07-08 02:36:36,038] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
